AY2021S2-CS2103-T14-1/tp
# Fix add issue and edit issue commands

## What this does
This PR:
1. Forces add and edit issue to only accept existing rooms

Fixes #145 

## How to test

Test Command
1. Run SunRez
2. Ensure rooms 10-100 and 12-110 exists first
2. Execute `iadd r/10-100 d/Broken light`
3. It should succeed
4. Execute `iadd r/50-100 d/Broken light`
5. It should say "There is no such room"
6. Execute `iedit 1 r/50-100`
7. It should say "There is no such room"
8. Execute `iedit 1 r/12-110`
9. It should succeed

Run automated Tests
1. Run tests
2. Check that they all pass
